addon/components/date-range-picker.js

```diff
@@ -110,6 +118,8 @@ export default Ember.Component.extend({
       endDate: endDate,
       minDate: minDate,
       maxDate: maxDate,
+      isInvalidDate: this.isInvalidDate.bind(this.context),
```

Comment:
Not actually because it does nothing by default, it's just a hook. About the `this.context` bind, I had to add that so we can have an outside context to work with, otherwise you'll be on the plugin scope with nothing to check against and decide if something is a valid date o not.